gradle/project.gradle.kts

```diff
@@ -239,4 +252,7 @@ application {
             "--add-opens", "javafx.web/com.sun.webkit=ALL-UNNAMED",
             "-Duser.dir=$workingDir"
     )
-}
\ No newline at end of file
+}
+
+if (JavaVersion.current()!=JavaVersion.VERSION_1_9)
```

Comment:
the script is first compiled, and then the actual requested task is run. This already executes at compile time, so it fails even before it tries to run any task. I just didn't want it to visually get in the way, since it's only a safety check.